os/audio/audio.c

```diff
@@ -990,7 +991,7 @@ int audio_register(FAR const char *name, FAR struct audio_lowerhalf_s *dev)
 
 			ret = mkdir(path, 0644);
 			if (ret < 0) {
-				auddbg("ERROR: mkdir failed\n");
+				auddbg("%s %s \n", clog_message_str[CMN_LOG_FILE_OPEN_ERROR], path);
```

Comment:
This is not file open error. Can we add new `CMN_LOG_DIR_CREATE_ERROR`?

Reply:
@NayanaNagaraj23 you can operation fail message.

Reply:
OK, I will change it.